### DISCUSSION TITLE
How do I construct a categorical distribution?

### DISCUSSION THREAD

**Discussion by irisallevi** [cscherrer/MeasureTheory.jl#268] [Q&A]
Hi,
I like this package, but I'm having some issues in reproducing some basics features of Distributions.jl. 

Maybe I'm missing something, but I didn't see a Categorical measure in the API. How do I define one? 

Of course a proxy would be to consider a Multinomial with n=1, but this would store a vector of k elements with only one non zero entry, which is useless.

Thanks in advance!

**Comment by mschauer** [comment]
See #148 , there is a pending pull request.

**Reply by irisallevi** [reply]
Thanks for this.
Is this pull request from 2021? It seems quite dead, doesn't it?
